package.json

```diff
@@ -46,6 +46,7 @@
   "devDependencies": {
     "@types/lodash.debounce": "4.0.3",
     "@types/prop-types": "15.5.2",
-    "@types/react-native": "0.49.5"
+    "@types/react-native": "0.49.5",
+    "@types/react": "16.4.7"
```

Comment:
We should provide a specific version of react types as react native types has them as dependencies. The build was breaking because it pointed to the latest version of types/react.